package.json

```diff
@@ -47,17 +47,19 @@
     "insert-css": "2.0.0",
     "lint-staged": "^8.1.5",
     "lodash.once": "^4.0.0",
-    "mapbox-gl": "^0.27.0",
+    "mapbox-gl": "^0.53.0",
```

Comment:
To confirm, do we want mapbox-gl as a devDep and peerDep? I'm not well versed on how peerDep work so just want to gut check in case this was in error.

Reply:
I believe we want it to be both. During regular use, we want it as a peer dependency because we expect the plugin to be used in conjunction with a mapbox-gl map. However, during development we want it as a dev dependency so we can use it in tests.

Reply:
Is v0.53.0 the minimum requirement? I think if possible we should try to set this to the minimum version required for everything to work in gl-geocoder.

Not essential, just a nice to have.